.tox-coveragerc

```diff
@@ -1,14 +0,0 @@
-[run]
-branch = True
-source =
-    hyperlink
-    ../hyperlink
-omit =
-    */flycheck_*
```

Comment:
I may want to get this omit directive back just for me and my fellow emacs flycheck users. :)

Reply:
Ah.  I had no idea what that was for, and got it in my head that it must be some leftover thing and then forgot to ask.  I'll stick it back in.

Reply:
done